### PR TITLE
[#3234] - move select all checkbox to accordion

### PR DESCRIPTION
**Pull request by marvinkome**
## Checklist
* [x] Connect the issue
* [ ] Test plan
* [ ] Copyright header
* [x] Code formatting
* [ ] Documentation

### CODE REVIEW

File: Dashboard/app/js/lib/views/devices/assignment-edit-views.jsx [hunk 1]
```diff
@@ -357,7 +357,9 @@ FLOW.AssignmentEditView = FLOW.ReactComponentView.extend(
             this.deviceGroups[dgId] = {
               0: {
                 name: 'Select all devices',
-                checked: numberOfDevices === numberOfSelectedDevices,
+                checked:
+                  numberOfDevices !== 0 &&
+                  numberOfDevices === numberOfSelectedDevices,
```
Comment (muloem) [inline Dashboard/app/js/lib/views/devices/assignment-edit-views.jsx]: Wouldn't the second condition by itself suffice?

**Comment by muloem** [comment]
I think we can consider adding this for the next release round.

**Comment by janagombitova** [comment]
I agree. Let's get the already tested batch of changes out and push this to the next round

File: Dashboard/app/js/lib/views/devices/assignment-edit-views.jsx (hunk 1 above)
Comment (marvinkome) [reply Dashboard/app/js/lib/views/devices/assignment-edit-views.jsx]: No, the first to make sure the select-box isn't checked when they're no devices in the group

**Pull request merged**